packages/client/__tests__/Authentication.test.tsx

```diff
@@ -458,31 +458,71 @@ const authenticationModelMachine =
-                                                expect(
-                                                    serverErrorAlert,
-                                                ).toBeTruthy();
+                                                    await waitFor(() => {
+                                                        const serverErrorAlert =
+                                                            within(
+                                                                screen.getByTestId(
+                                                                    'signing-in-screen-server-error',
+                                                                ),
+                                                            ).getByRole(
+                                                                'alert',
+                                                            );
+                                                        expect(
+                                                            serverErrorAlert,
+                                                        ).toBeTruthy();
 
-                                                expect(
-                                                    serverErrorAlert,
-                                                ).toHaveTextContent(
-                                                    'Credentials are invalid',
-                                                );
-                                            });
+                                                        expect(
+                                                            serverErrorAlert,
+                                                        ).toHaveTextContent(
+                                                            'Credentials are invalid',
+                                                        );
+                                                    });
+                                                },
+                                            },
+                                        },
+                                        'Unknown server error': {
+                                            meta: {
+                                                test: async ({
+                                                    screen,
+                                                }: TestingContext) => {
+                                                    invariant(
+                                                        screen !== undefined,
+                                                        'Screen must have been rendered',
+                                                    );
+
+                                                    await waitFor(() => {
+                                                        const serverErrorAlert =
+                                                            within(
+                                                                screen.getByTestId(
```

Comment:
Not important at all but on the sign up screen we do display a toast error, maybe we should pick at least one common solution for both screens ?
Might be later

Reply:
Indeed, we will have to normalize the UX of the authentication screens.
I did not want to take these decisions alone so I went to the simplest solution, according to the code I already had.

Reply:
We will then determine all of this details later during our UI/UX improvements journey